BillTrack.Api/Program.cs

```diff
@@ -1,16 +1,25 @@
+global using FluentValidation;
 using BillTrack.Api.Configurations;
 using FastEndpoints;
 
 var builder = WebApplication.CreateBuilder(args);
 
-//builder.Services.AddFastEndpoints();
+builder.Services.AddFastEndpoints();
+builder.Logging.AddConsole();
 
 builder.Services
     .ConfigureInterceptors()
-    .ConfigureDatabase(builder.Configuration);
+    .ConfigureDatabase(builder.Configuration)
+    .ConfigureRepositories()
+    .ConfigureServices()
+    .ConfigureMappers();
 
 var app = builder.Build();
 
-//app.UseFastEndpoints();
+app.UseExceptionHandler(_ => { });
+app.UseFastEndpoints(c =>
+{
+    c.Endpoints.RoutePrefix = "api";
```

Comment:
if you setup the prefix, then it could include a version sign. To host a few versions of an endpoint simultaneously.
For example
```
c.Endpoints.RoutePrefix = "api/v1"
```